<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_Game">
    <property name="title">
     <string>New Game</string>
    </property>
    <addaction name="actionAs_Server"/>
    <addaction name="actionAs_Client"/>
    <addaction name="actionLeave_Game"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuNew_Game"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAs_Server"/>
   <addaction name="separator"/>
   <addaction name="actionAs_Client"/>
   <addaction name="separator"/>
   <addaction name="actionLeave_Game"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionAs_Server">
   <property name="text">
    <string>As Server</string>
   </property>
  </action>
  <action name="actionAs_Client">
   <property name="text">
    <string>As Client</string>
   </property>
  </action>
  <action name="actionLeave_Game">
   <property name="text">
    <string>Leave Game</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
